src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 600;
+    private static final int DEFAULT_RADIUS = 10;
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+    private static final String COLOR = colorSupplier.getRandomColor();
```

Comment:
you don't generate a new color for each call of the method, this constant is set once upon class creation, and doesn't change in runtime